src/matcha_ml/cli/cli.py

```diff
@@ -110,10 +116,17 @@ def get(
 
 @app.command()
 @track(event_name=AnalyticsEvent.DESTROY)
-def destroy() -> None:
-    """Destroy the provisioned cloud resources. It will destroy the resource group even if resources are provisioned inside the group."""
+def destroy(
+    full: Optional[str] = typer.Argument(None),
```

Comment:
Can we also add a descriptive help for this?

Reply:
The `help` argument is not passed to full typer.Argument. It should provide a short description of what `full` option is doing.